### PR TITLE
CEDS-2455 - Move the 'wrong Enrollment' page to GDS

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -62,6 +59,10 @@ class AppConfig @Inject()(
   lazy val classificationHelpUrl = loadConfig("urls.classificationHelp")
   lazy val ecicsToolUrl = loadConfig("urls.ecicsTool")
 
+  lazy val eoriService = loadConfig("urls.eoriService")
```
Comment (SteveSugden) [inline app/config/AppConfig.scala:62]: Should I move these (and other "static" urls.xxx values) into it's own "config" class?  E.g. ExternalLinksConfig.

Comment (PatrykRudnicki) [reply app/config/AppConfig.scala:62]: I think that can be a good idea, but it's up to you :)

File: test/views/UnauthorisedViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+@ViewTest
+class UnauthorisedViewSpec extends UnitViewSpec with Stubs with Injector {
+
+  private val appConfig = mock[AppConfig]
+  when(appConfig.customsDeclarationsGoodsTakenOutOfEuUrl)
+    .thenReturn("appConfig.customsDeclarationsGoodsTakenOutOfEuUrl")
+  when(appConfig.commodityCodesUrl).thenReturn("appConfig.commodityCodesUrl")
+  when(appConfig.relevantLicensesUrl).thenReturn("appConfig.relevantLicensesUrl")
+  when(appConfig.serviceAvailabilityUrl).thenReturn("appConfig.serviceAvailabilityUrl")
+  when(appConfig.customsMovementsFrontendUrl).thenReturn("appConfig.customsMovementsFrontendUrl")
```
Comment (PatrykRudnicki) [inline test/views/UnauthorisedViewSpec.scala:37]: This `appConfig` is unnecessary.

You're not using it anywhere so this part can be removed :)

Comment (SteveSugden) [reply test/views/UnauthorisedViewSpec.scala]: Yes.  I copied this from another view spec.  Good spot.

**Pull request merged**